admin/assets/stylesheets/elements/_sorting.scss

```diff
@@ -0,0 +1,13 @@
+.sort {
+  display: inline-block;
+  width: 0;
+  height: 0;
+  border-style: solid;
+  border-width: 12px 6px 0 6px;
+  border-color: $govuk-blue transparent transparent transparent;
+}
+
+.sort.up {
+  border-width: 0 6px 12px 6px;
+  border-color: transparent transparent $govuk-blue transparent;
+}
```

Comment:
might have to remove my sorting css when this is merged I suppose. Didn't the arrow get too close to the text with your implementation?

Reply:
@GeorgeChatzigiannis I solved the space on the left separately to keep the sorting arrows as generic and simple as possible